### ISSUE TITLE
failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use

### ISSUE THREAD

**Issue by farostegui** [code-ready/crc#2613]
When trying to start crc locally I am getting the following error:
```
franciscoarostegui@Franciscos-MacBook-Pro crc_hyperkit_4.7.18 % crc start
INFO Checking if running as non-root              
INFO Checking if crc-admin-helper executable is cached 
INFO Checking for obsolete admin-helper executable 
INFO Checking if running on a supported CPU architecture 
INFO Checking minimum RAM requirements            
INFO Checking if running emulated on a M1 CPU     
INFO Checking if HyperKit is installed            
INFO Checking if qcow-tool is installed           
INFO Checking if crc-driver-hyperkit is installed 
INFO Starting CodeReady Containers VM for OpenShift 4.7.18... 
failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use
franciscoarostegui@Franciscos-MacBook-Pro crc_hyperkit_4.7.18 % 
```

**Comment by adrianriobo** [comment]
Can you check if port 443 is already in use?

`sudo lsof -i:443`

**Comment by farostegui** [comment]
Hello Adrian,

I hope you are doing well. Thank you for your help on this issue. As per your request below information solicited. If you have any further questions do not hesitate to call me.


COMMAND     PID               USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME

httpd       122               root    6u  IPv6 0xa8924255b9a3b77f      0t0  TCP *:https (LISTEN)

iCoreServ   142               root   14u  IPv6 0xa8924255de3a277f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54794->g2600-1403-9c00-0d83-0000-0000-0000-0aeb.deploy.static.akamaitechnologies.com:https (ESTABLISHED)

parsecd     594 franciscoarostegui    8u  IPv4 0xa8924255c39928d7      0t0  TCP 10.0.0.93:56274->ec2-54-236-229-12.compute-1.amazonaws.com:https (ESTABLISHED)

OneDrive    700 franciscoarostegui   14u  IPv4 0xa8924255d2c911bf      0t0  TCP 10.0.0.93:55181->52.242.211.89:https (ESTABLISHED)

OneDrive    700 franciscoarostegui   15u  IPv4 0xa8924255d2c911bf      0t0  TCP 10.0.0.93:55181->52.242.211.89:https (ESTABLISHED)

Google      711 franciscoarostegui   22u  IPv6 0xa8924255de524abf      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54726->[2001:558:feed:443::5]:https (ESTABLISHED)

Google      711 franciscoarostegui   23u  IPv6 0xa8924255de4f045f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54724->[2001:558:feed:443::5]:https (ESTABLISHED)

Google      711 franciscoarostegui   25u  IPv6 0xa8924255de52511f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54728->[2001:558:feed:443::5]:https (ESTABLISHED)

Google      711 franciscoarostegui   32u  IPv6 0xa8924255bd4bc11f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:55671->mia07s61-in-x0e.1e100.net:https (CLOSED)

Google      711 franciscoarostegui   40u  IPv6 0xa8924255de3bf11f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54770->[2001:559:19:e000::b854:f463]:https (CLOSED)

Google      711 franciscoarostegui   64u  IPv6 0xa8924255de3bddff      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54836->[2001:559:19:e000::b854:f463]:https (CLOSED)

httpd       896               _www    6u  IPv6 0xa8924255b9a3b77f      0t0  TCP *:https (LISTEN)

Microsoft  2156 franciscoarostegui   44u  IPv6 0xa8924255de41bdff      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54708->[2603:1036:303:3c1d::2]:https (ESTABLISHED)

Microsoft  2156 franciscoarostegui   45u  IPv6 0xa8924255de41cabf      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54709->[2603:1036:303:3c1d::2]:https (ESTABLISHED)

Microsoft  2156 franciscoarostegui   49u  IPv6 0xa8924255bd4bbabf      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54711->[2603:1036:303:3c1d::2]:https (ESTABLISHED)

Microsoft  2156 franciscoarostegui   50u  IPv6 0xa8924255c50b9dff      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54712->[2603:1036:303:3c1d::2]:https (ESTABLISHED)

Microsoft  2156 franciscoarostegui   51u  IPv6 0xa8924255b9a3b11f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54713->[2603:1036:303:3c1d::2]:https (ESTABLISHED)

Microsoft  2156 franciscoarostegui   54u  IPv6 0xa8924255de7f0dff      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54715->[2603:1036:303:3c1d::2]:https (ESTABLISHED)

StocksWid  7231 franciscoarostegui   13u  IPv4 0xa8924255b8ab3be7      0t0  TCP 10.0.0.80:53014->media-router-apple71.prod.media.vip.bf1.yahoo.com:https (CLOSED)

StocksWid  7231 franciscoarostegui   18u  IPv6 0xa8924255de41c45f      0t0  TCP [2601:589:c200:a300:e02a:c6af:e9a7:2519]:53016->g2600-1403-8400-0193-0000-0000-0000-3efe.deploy.static.akamaitechnologies.com:https (CLOSED)

Microsoft 12057 franciscoarostegui   21u  IPv6 0xa8924255de52445f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54725->[2001:558:feed:443::5]:https (ESTABLISHED)

Microsoft 12057 franciscoarostegui   22u  IPv6 0xa8924255de52577f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54727->[2001:558:feed:443::5]:https (ESTABLISHED)

Microsoft 12057 franciscoarostegui   25u  IPv6 0xa8924255de33011f      0t0  TCP [2601:589:c200:a300:28a0:1263:63c3:1640]:54735->[2001:558:feed:443::5]:https (ESTABLISHED)

VirtualBo 53646 franciscoarostegui  122u  IPv4 0xa8924255c18711bf      0t0  TCP 10.0.0.80:53095->ec2-35-83-171-77.us-west-2.compute.amazonaws.com:https (ESTABLISHED)

Best regards,

Francisco Arostegui
ACDBE DBE SBE-G&S HABE MICRO MWBE MBE SDB DoTC HAO MOB
AFL INTERNATIONAL CONSULTING STAFF AND SERVICES INC.
Cell: (786) 332-8095
https://www.aflicss.com<https://www.aflicss.com/> | https://www.aflicss.com.co<https://www.aflicss.com.co/>
[LogoTransparente][signature_2036033975]<https://www.aflicss.com/trendmicro.html>  [Diagram  Description automatically generated with medium confidence] <https://www.youracclaim.com/badges/f8388064-f91f-47a6-997f-99b593ebd5a2/linked_in>    [A picture containing text, compact disk  Description automatically generated] <https://www.credly.com/badges/7db07025-60fb-4265-8fb7-e8e6109a3528/public_url>    [Diagram, venn diagram  Description automatically generated] <https://www.credly.com/badges/e7945455-9f4c-4287-95cb-68f51f2ad457/public_url>
 This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.



From: Adrián Riobo Lorenzo ***@***.***>
Date: Wednesday, July 28, 2021 at 4:31 AM
To: code-ready/crc ***@***.***>
Cc: FRANCISCO AROSTEGUI ***@***.***>, Author ***@***.***>
Subject: Re: [code-ready/crc] failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use (#2613)

Can you check if port 443 is already in use?

sudo lsof -i:443

**Comment by guillaumerose** [comment]
You already have a server running on the https port. Can you please stop it and start again crc ?

Thanks

**Comment by farostegui** [comment]
Hello Rose,

Thank you for all your help and support, that was my issues, I was running apache in port 443. May I know if there is a way, I could change port 443 to 444 in crc when I am creating a new container? If so, may I Know how? Thank you.

Best regards,

Francisco Arostegui
ACDBE DBE SBE-G&S HABE MICRO MWBE MBE SDB DoTC HAO MOB
AFL INTERNATIONAL CONSULTING STAFF AND SERVICES INC.
Cell: (786) 332-8095
https://www.aflicss.com<https://www.aflicss.com/> | https://www.aflicss.com.co<https://www.aflicss.com.co/>
[LogoTransparente][signature_2036033975]<https://www.aflicss.com/trendmicro.html>  [Diagram  Description automatically generated with medium confidence] <https://www.youracclaim.com/badges/f8388064-f91f-47a6-997f-99b593ebd5a2/linked_in>    [A picture containing text, compact disk  Description automatically generated] <https://www.credly.com/badges/7db07025-60fb-4265-8fb7-e8e6109a3528/public_url>    [Diagram, venn diagram  Description automatically generated] <https://www.credly.com/badges/e7945455-9f4c-4287-95cb-68f51f2ad457/public_url>
 This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.



From: Guillaume Rose ***@***.***>
Date: Wednesday, July 28, 2021 at 8:14 AM
To: code-ready/crc ***@***.***>
Cc: FRANCISCO AROSTEGUI ***@***.***>, Author ***@***.***>
Subject: Re: [code-ready/crc] failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use (#2613)

You already have a server running on the https port. Can you please stop it and start again crc ?

Thanks

**Comment by farostegui** [comment]
Hello Rose,

One more thing if possible. I do have Trend Micro IWSaaS Enforcement agent which If I have it active with my WI-FI. If I start CRC Open Web Console does not work but if I disable my WI-FI it works. I append all IP involved in my proxy to bypass them but is not working. Could you please help me on this or should I open a new ticket? Please advise. Thank you.

Without WI-FI

***@***.***

With WI-FI active

***@***.***

Proxy Bypassed

***@***.***
Best regards,

Francisco Arostegui
ACDBE DBE SBE-G&S HABE MICRO MWBE MBE SDB DoTC HAO MOB
AFL INTERNATIONAL CONSULTING STAFF AND SERVICES INC.
Cell: (786) 332-8095
https://www.aflicss.com<https://www.aflicss.com/> | https://www.aflicss.com.co<https://www.aflicss.com.co/>
[LogoTransparente][signature_2036033975]<https://www.aflicss.com/trendmicro.html>  [Diagram  Description automatically generated with medium confidence] <https://www.youracclaim.com/badges/f8388064-f91f-47a6-997f-99b593ebd5a2/linked_in>    [A picture containing text, compact disk  Description automatically generated] <https://www.credly.com/badges/7db07025-60fb-4265-8fb7-e8e6109a3528/public_url>    [Diagram, venn diagram  Description automatically generated] <https://www.credly.com/badges/e7945455-9f4c-4287-95cb-68f51f2ad457/public_url>
 This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.



From: ***@***.*** ***@***.***>
Date: Wednesday, July 28, 2021 at 11:09 AM
To: code-ready/crc ***@***.***>, code-ready/crc ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [code-ready/crc] failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use (#2613)
Hello Rose,

Thank you for all your help and support, that was my issues, I was running apache in port 443. May I know if there is a way, I could change port 443 to 444 in crc when I am creating a new container? If so, may I Know how? Thank you.

Best regards,

Francisco Arostegui
ACDBE DBE SBE-G&S HABE MICRO MWBE MBE SDB DoTC HAO MOB
AFL INTERNATIONAL CONSULTING STAFF AND SERVICES INC.
Cell: (786) 332-8095
https://www.aflicss.com<https://www.aflicss.com/> | https://www.aflicss.com.co<https://www.aflicss.com.co/>
[LogoTransparente][signature_2036033975]<https://www.aflicss.com/trendmicro.html>  [Diagram  Description automatically generated with medium confidence] <https://www.youracclaim.com/badges/f8388064-f91f-47a6-997f-99b593ebd5a2/linked_in>    [A picture containing text, compact disk  Description automatically generated] <https://www.credly.com/badges/7db07025-60fb-4265-8fb7-e8e6109a3528/public_url>    [Diagram, venn diagram  Description automatically generated] <https://www.credly.com/badges/e7945455-9f4c-4287-95cb-68f51f2ad457/public_url>
 This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.



From: Guillaume Rose ***@***.***>
Date: Wednesday, July 28, 2021 at 8:14 AM
To: code-ready/crc ***@***.***>
Cc: FRANCISCO AROSTEGUI ***@***.***>, Author ***@***.***>
Subject: Re: [code-ready/crc] failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use (#2613)

You already have a server running on the https port. Can you please stop it and start again crc ?

Thanks

**Comment by guillaumerose** [comment]
> May I know if there is a way, I could change port 443 to 444 in crc when I am creating a new container?

No, you can't change that easily sorry.

> I append all IP involved in my proxy to bypass them but is not working

You can try to add `.crc.testing` domain to your proxy bypass list.
Console and apiserver are listenning on 127.0.0.1 so it should not be blocked by your firewall.

**Comment by farostegui** [comment]
Hello Rose,

Thank you for all your help and support. I did what you mentioned but still getting same issue. Please see screens shot below. After I applied latest update, I am getting the following message:


INFO Operator image-registry is degraded

Please your help. Thank you.

***@***.***

***@***.***

If I run it with the oc command I am ok, see screen shot below:

***@***.***

Best regards,

Francisco Arostegui
ACDBE DBE SBE-G&S HABE MICRO MWBE MBE SDB DoTC HAO MOB
AFL INTERNATIONAL CONSULTING STAFF AND SERVICES INC.
Cell: (786) 332-8095
https://www.aflicss.com<https://www.aflicss.com/> | https://www.aflicss.com.co<https://www.aflicss.com.co/>
[LogoTransparente][signature_2036033975]<https://www.aflicss.com/trendmicro.html>  [Diagram  Description automatically generated with medium confidence] <https://www.youracclaim.com/badges/f8388064-f91f-47a6-997f-99b593ebd5a2/linked_in>    [A picture containing text, compact disk  Description automatically generated] <https://www.credly.com/badges/7db07025-60fb-4265-8fb7-e8e6109a3528/public_url>    [Diagram, venn diagram  Description automatically generated] <https://www.credly.com/badges/e7945455-9f4c-4287-95cb-68f51f2ad457/public_url>
 This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.



From: Guillaume Rose ***@***.***>
Date: Thursday, July 29, 2021 at 8:27 AM
To: code-ready/crc ***@***.***>
Cc: FRANCISCO AROSTEGUI ***@***.***>, Author ***@***.***>
Subject: Re: [code-ready/crc] failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use (#2613)

May I know if there is a way, I could change port 443 to 444 in crc when I am creating a new container?

No, you can't change that easily sorry.

I append all IP involved in my proxy to bypass them but is not working

You can try to add .crc.testing domain to your proxy bypass list.
Console and apiserver are listenning on 127.0.0.1 so it should not be blocked by your firewall.

**Comment by farostegui** [comment]
INFO Loading bundle: crc_hyperkit_4.8.5...        
INFO Creating CodeReady Containers VM for OpenShift 4.8.5... 
INFO Generating new SSH Key pair...               
INFO Generating new password for the kubeadmin user 
INFO Starting CodeReady Containers VM for OpenShift 4.8.5... 
failed to expose port :80 -> 192.168.127.2:80: listen tcp :80: bind: address already in use
franciscoarostegui@Franciscos-MacBook-Pro ~ % sudo lsof -i:80 
Password:
COMMAND   PID               USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
httpd     119               root    4u  IPv6 0xce072bf82c0c8971      0t0  TCP *:http (LISTEN)
iCoreServ 138               root   15u  IPv4 0xce072bf82a2c0701      0t0  TCP 10.0.0.94:50250->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
iCoreServ 138               root   17u  IPv4 0xce072bf82a2bfcd9      0t0  TCP 10.0.0.94:50255->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 448 franciscoarostegui   34u  IPv4 0xce072bf8309a5129      0t0  TCP 10.0.0.94:50055->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 448 franciscoarostegui   35u  IPv4 0xce072bf828338cd9      0t0  TCP 10.0.0.94:50049->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 448 franciscoarostegui   56u  IPv4 0xce072bf82b777a11      0t0  TCP 10.0.0.94:50062->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 448 franciscoarostegui   65u  IPv4 0xce072bf830947a11      0t0  TCP 10.0.0.94:50080->ec2-52-90-127-255.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 448 franciscoarostegui   98u  IPv4 0xce072bf8299f7a11      0t0  TCP 10.0.0.94:50147->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
NewsToday 504 franciscoarostegui   13u  IPv4 0xce072bf820328579      0t0  TCP 10.0.0.94:50036->ec2-52-90-127-254.compute-1.amazonaws.com:http (CLOSED)
Microsoft 551 franciscoarostegui   23u  IPv4 0xce072bf82c0b8579      0t0  TCP 10.0.0.94:49985->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Microsoft 551 franciscoarostegui   24u  IPv4 0xce072bf83094acd9      0t0  TCP 10.0.0.94:49990->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Microsoft 551 franciscoarostegui   25u  IPv4 0xce072bf82c86d439      0t0  TCP 10.0.0.94:50180->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 551 franciscoarostegui   27u  IPv4 0xce072bf82b8fb701      0t0  TCP 10.0.0.94:50009->ec2-52-90-127-255.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 551 franciscoarostegui   29u  IPv4 0xce072bf82c0b2a11      0t0  TCP 10.0.0.94:50019->ec2-52-90-127-255.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 551 franciscoarostegui   30u  IPv4 0xce072bf820328fa1      0t0  TCP 10.0.0.94:50181->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
Google    646 franciscoarostegui   26u  IPv4 0xce072bf82a2bd439      0t0  TCP 10.0.0.94:49987->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Google    646 franciscoarostegui   27u  IPv4 0xce072bf8309a4701      0t0  TCP 10.0.0.94:49989->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Google    646 franciscoarostegui   28u  IPv4 0xce072bf82e5cc579      0t0  TCP 10.0.0.94:50326->ec2-52-90-127-254.compute-1.amazonaws.com:http (ESTABLISHED)
Microsoft 651 franciscoarostegui   18u  IPv4 0xce072bf82c86f2b1      0t0  TCP 10.0.0.94:49982->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Microsoft 651 franciscoarostegui   23u  IPv4 0xce072bf82cc6fa11      0t0  TCP 10.0.0.94:49983->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Microsoft 651 franciscoarostegui   26u  IPv4 0xce072bf83094a2b1      0t0  TCP 10.0.0.94:50004->ec2-52-90-127-255.compute-1.amazonaws.com:http (ESTABLISHED)
Adobe\x20 693 franciscoarostegui   25u  IPv4 0xce072bf8309a5b51      0t0  TCP 10.0.0.94:49984->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Adobe\x20 693 franciscoarostegui   27u  IPv4 0xce072bf82c870701      0t0  TCP 10.0.0.94:49988->s3-us-west-2-w.amazonaws.com:http (CLOSED)
Core\x20S 752 franciscoarostegui   32u  IPv4 0xce072bf82b01b439      0t0  TCP 10.0.0.94:50309->ec2-52-90-127-255.compute-1.amazonaws.com:http (ESTABLISHED)
Core\x20S 752 franciscoarostegui   33u  IPv4 0xce072bf82b01b439      0t0  TCP 10.0.0.94:50309->ec2-52-90-127-255.compute-1.amazonaws.com:http (ESTABLISHED)
httpd     803               _www    4u  IPv6 0xce072bf82c0c8971      0t0  TCP *:http (LISTEN)

**Comment by cfergeau** [comment]
Probably the same issue as https://github.com/code-ready/crc/issues/2760

**Comment by cfergeau** [comment]
Duplicate of #2760

**Comment by saschpe** [comment]
The same issue occurs on macOS when Podman is running:

```shell
$ crc start 
INFO Using bundle path /Users/saschpe/.crc/cache/crc_vfkit_4.21.4_arm64.crcbundle 
... 
ERRO failed to expose port :443 -> 192.168.127.2:443: listen tcp :443: bind: address already in use 

$ lsof -i:80                                                                                                                                               git:main*
COMMAND  PID    USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
gvproxy 5375 saschpe   29u  IPv6 0x74108066b71623ca      0t0  TCP *:http (LISTEN)

$ podman machine stop default                                                                                                                              
Machine "default" stopped successfully

$ lsof -i:80                                                                                                                                               

$ crc start           
INFO Using bundle path /Users/saschpe/.crc/cache/crc_vfkit_4.21.4_arm64.crcbundle 
...
INFO Loading bundle: crc_vfkit_4.21.4_arm64...    
INFO Starting CRC VM for openshift 4.21.4...      
INFO CRC instance is running with IP 127.0.0.1    
INFO CRC VM is running                            
INFO Updating authorized keys...                  
INFO Configuring shared directories               
INFO Check internal and public DNS query...   
```

So not really (only) a duplicate of a Windows issue.
